src/bootstrap/bin/rustc.rs

```diff
@@ -94,6 +94,10 @@ fn main() {
         // other crate intentionally as this is the only crate for now that we
         // ship with panic=abort.
         //
+        // FIXME: Remove the special case for "panic_abort"
+        // once https://github.com/rust-lang/rust/pull/60026
+        // rides the release train into the bootstrap compiler.
```

Comment:
Could you stick a `cfg(bootstrap)` literally in this comment? That way it's likely we'll notice this when doing the bootstrap upgrade :)

Reply:
Added